controllers/operands/kubevirtConsolePlugin_test.go

```diff
@@ -344,6 +344,159 @@ var _ = Describe("Kubevirt Console Plugin", func() {
 			// let's check the object UID to ensure that the object get really deleted and recreated
 			Expect(foundResource.GetUID()).ToNot(Equal(types.UID("oldObjectUID")))
 		})
+
+		Context("Node Placement", func() {
+
+			It("should add node placement if missing", func() {
+				existingResource, err := NewKvUIPluginDeplymnt(hco)
+				Expect(err).ToNot(HaveOccurred())
+
+				hco.Spec.Workloads.NodePlacement = commontestutils.NewNodePlacement()
+				hco.Spec.Infra.NodePlacement = commontestutils.NewOtherNodePlacement()
+
+				cl := commontestutils.InitClient([]client.Object{hco, existingResource})
+				handler, err := newKvUIPluginDeploymentHandler(logger, cl, commontestutils.GetScheme(), hco)
+				Expect(err).ToNot(HaveOccurred())
+				res := handler[0].ensure(req)
+				Expect(res.Created).To(BeFalse())
+				Expect(res.Updated).To(BeTrue())
+				Expect(res.Overwritten).To(BeFalse())
+				Expect(res.UpgradeDone).To(BeFalse())
+				Expect(res.Err).ToNot(HaveOccurred())
+
+				foundResource := &appsv1.Deployment{}
+				Expect(
+					cl.Get(context.TODO(),
+						types.NamespacedName{Name: existingResource.Name, Namespace: existingResource.Namespace},
+						foundResource),
+				).ToNot(HaveOccurred())
+
+				Expect(existingResource.Spec.Template.Spec.NodeSelector).To(BeZero())
+				Expect(existingResource.Spec.Template.Spec.Affinity).To(BeZero())
+				Expect(existingResource.Spec.Template.Spec.Tolerations).To(BeZero())
```

Comment:
what is the meaning of, for example, nodeSelector == 0? is it a number?

Maybe it should be:
```golang
Expect(existingResource.Spec.Template.Spec.NodeSelector).To(BeEmpty())
```